installer/README.md

```diff
@@ -131,28 +173,12 @@ $ installer/install
 
 Note: Multiple instances of exporter and api is not allowed currently.
 
-#### Post install verification
-After delfin installation use the following command to verify all process 
-of delfin are running.
-```sh
-ps -aux | grep delfin
-```
-
-# Uninstall
-Running the uninstall script will stop all delfin processes and do cleanup
-```sh
-installer/uninstall
-
-# Example
-root@root1:~/delfin-demo/delfin$ installer/uninstall
-```
-
-# Logs
+### Logs
 All the installer logs are stored in the /var/log/sodafoundation directory.
```

Comment:
can we change the directory name from sodafoundation to soda?

Reply:
Done